app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java

```diff
@@ -210,4 +246,22 @@ public boolean crfToolbarShowProgress() {
     public boolean controlMediaVolume() {
         return crfInstructionStep.mediaVolume;
     }
+
+    public void remindMeLater() {
```

Comment:
I don't think the reminders will actually get scheduled without handling the activity result.  Look at CrfStartTaskStepLayout for an example, or could CrfSkipInstructionStepLayout just extend CrfStartTaskStepLayout?